ZichanYang324/BF-Employee-HR
# BGP48 - [Frontend and backend] - Fix bugs for employee part and add connection to all pages

# Description
Fix all the bugs and add the connection to all pages

## Related PRS (if any):
https://github.com/ZichanYang324/BF-Employee-HR/pull/25
https://github.com/ZichanYang324/BF-Employee-HR/pull/32



## Main changes explained:
1. create login page in HR, and add route protector, check http://localhost:4200/login
2. after user login, user will be routed to Onboard or Info by application status
3. fix Yu's issue in Name section
4. fix Christol's issue about unable to upload onboarding data&files, and correct the data upload&retreive from AWS s3
5. add Navbar into App.jsx into Client UI, and connect all pages into Navbar
6.  add Visa status files filter: we don't need to show profile pic in visa documents
7. add profile pic into nav bar

## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally

## Screenshots or videos of changes or expecting result:
will show this part in our meeting


## Note:
Include the information the reviewers need to know.
